src/tests/SeleniumBrowser.php

```diff
@@ -0,0 +1,283 @@
+<?php
```

Comment:
Based a lot upon our platform [BrowserHelper](https://github.com/orgnc/platform/blob/c880cf2da990b05b8fd1472eb8b47f35d640a63e/backend/tests/base.py#L155).

Reply:
I'm not _thrilled_ about some of the selectors I've used for WP-specific elements, but since I don't have control over what WordPress does, I did my best.